subprojects/gdk_rust/gdk_common/src/model.rs

```diff
@@ -312,6 +326,20 @@ pub struct TxListItem {
     pub transaction_weight: usize,
 }
 
+// Negative (sent) amounts are expceted to be provided as positive numbers.
```

Comment:
nit: `expected`

Reply:
Fixed in https://github.com/Blockstream/gdk/pull/119/commits/cd2e9cd1ad35823b9f6b7143835c0165597704b8.